act-rules-format.bs

```diff
@@ -110,42 +110,90 @@ An ACT Rule MUST have a description that is in plain language, and provides a br
 </div>
 
-One reason why an [=atomic rule=] might not list accessibility requirements is because it is part of a [=composite rule=], and only a `failed` [=outcome=] in the composite rule would mean not satisfying the accessibility requirement. Another reason might be that an ACT Rule is designed to evaluate a best practice that is not part of any accessibility standard.
-
-Accessibility Requirements Mapping {#accessibility-requirements-mapping}
-------------------------------------------------------------------------
+<div class=example>
+  <p><strong>Example:</strong> Accessibility Requirements Mapping for a rule that tests if an image button has an accessible name:
+  <blockquote><ul>
+    <li><a href="https://www.w3.org/TR/WCAG21/#non-text-content">
+        Success Criterion 1.1.1: Non-text content
+      </a><ul>
+        <li><strong>Required for conformance</strong> to WCAG 2.0 and WCAG 2.1 level A</li>
+        <li>Outcome mapping: <ul>
+          <li>Failed outcome: not satisfied</li>
+          <li>Passed outcome: further testing is needed</li>
+          <li>Inapplicable outcome: further testing is needed</li>
+        </ul></li>
+      </ul>
+    </li>
+    <li><a href="https://www.w3.org/TR/WCAG21/#name-role-value">
+        Success Criterion 4.1.2: Name, Role, Value
+      </a><ul>
+        <li><strong>Required for conformance</strong> to WCAG 2.0 and WCAG 2.1 level A</li>
+        <li>Outcome mapping:<ul>
+          <li>Failed outcome: not satisfied</li>
+          <li>Passed outcome: further testing is needed</li>
+          <li>Inapplicable outcome: further testing is needed</li>
+        </ul></li>
+      </ul>
+    </li>
+  </ul></blockquote>
+</div>
 
-If any accessibility requirements are listed, the ACT Rule MUST indicate how each outcome impacts the conformance to the listed accessibility requirements. For WCAG 2.1 Success Criteria, this means indicating whether or not the success criterion can be satisfied, or if further testing is needed.
+ACT Rules can be used to test accessibility requirements that are not part of a W3C accessibility standard, such as WCAG advisory techniques, or a testing methodology like RGAA. When an ACT Rule is used to test something not required by its accompanying standard, the rule MUST indicate this as part of its accessibility requirements mapping. Examples of this are WCAG sufficient techniques, which are not required for WCAG conformance, or a company style guide which includes both requirements and optional "best practices". That distinction between what is required and what is optional has to be made clear.
```

Comment:
Should we have an explanation for what RGAA is? Or a link to it maybe?
And maybe the same for WCAG advisory techniques?

Reply:
Is it also a MUST to indicate when something IS required for conformance?